src/components/Contact/Contact.css

```diff
@@ -0,0 +1,52 @@
+/* src/components/Contact/Contact.css */
+.contact-section {
+  width: 100%;
+  min-height: 100vh;
+  background-color: #0A192F;
+  padding: 4rem 1rem;
```

Comment:
These styles wont be needed.

- there is default bg color (as per figma) added for main page in `src/App.css`